arch/cortex-m3/src/mpu.rs

```diff
@@ -410,7 +411,10 @@ impl kernel::mpu::MPU for MPU {
         }
 
         // Cortex-M regions can't be greater than 4 GB.
-        if math::log_base_two(region_size as u32) >= 32 {
+        if usize::try_from(u32::max_value())
```

Comment:
Since this is in an architecture specific crate, and we _know_ that `usize` on a cortex-m3 _is_ 32-bits, meaning there is no way to express more than 4GB... can we actually just remove this check?

Reply:
Yes. I'm updating the description too.